tests/Silverback.Integration.Kafka.Tests/Messaging/Configuration/Kafka/KafkaConsumerConfigTests.cs

```diff
@@ -25,6 +25,19 @@ public void IsAutoCommitEnabled_CorrectlySet(bool? enableAutoCommit, bool expect
             config.IsAutoCommitEnabled.Should().Be(expected);
         }
 
+        [Theory]
+        [InlineData("myGroup", true)]
+        [InlineData("group-id-not-set", false)]
```

Comment:
You should test the cases where the group id is set to null (maybe string empty as well?) and the case where it's not set at all.

Reply:
You're right, I've added some new test cases:
https://github.com/BEagle1984/silverback/pull/172/commits/79f770503b5fd550d24587292afcc5406b003e8d#diff-cd8bc913ddceb2c5d1a9de4849e14101b25e675822aa000712692b25dc71250b